src/commands/auth.ts

```diff
@@ -1,32 +1,120 @@
-import { open } from "openurl";
-import * as keytar from "keytar";
-import readline from "readline";
+import dotenv from 'dotenv';
+import open from "open";
+import readline from 'readline/promises';
+import fs from 'fs/promises';
+import path from 'path';
+import os from 'os';
+import crypto from 'crypto';
+import keytar from 'keytar';
+
+dotenv.config();
 
 const SERVICE_NAME = "AnyFlowCLI";
 const ACCOUNT_NAME = "apiToken";
+const ENCRYPTION_KEY = process.env.ENCRYPTION_KEY;
+
+if (!ENCRYPTION_KEY) {
```

Comment:
This is running at the moment you import auth.js and is preventing us of using the tool. This check is unnecessary as to get to the functions here you would need to pass through `main()` function which already checks the encryption key.

Steps to reproduce:
```
anyflow init
rm .env
anyflow auth
anyflow init
```

we get:

![image](https://github.com/user-attachments/assets/18b4907f-476d-412b-8b3f-391008476b6b)

impossible to init()

Reply:
Fixed, the problem was on the auth.ts file that made the verification outside the function